### DISCUSSION TITLE
v6 Docs - Upgrade

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#637] [Q&A (Help)] [answered]
I think you need to improve the "Update Guide" section a bit, I updated a couple of my apps and I report the problems I encountered:

1. Step 2 is missing the version of DevTools that would be useful
2. First of all you have to run the php artisan migrate:status command and be sure that there are no "pending" migrations because this creates problems later!
3. Step 3.2 doesn't say that the update must be done with two shells open, one for running the app and another for the update commands otherwise the basset verification fails! It should be reported..
4. Step 20 is wrong because when you get to this by converting a BackPack 5.x app, you already have a resources/views/vendor/backpack/ui folder which must be deleted before renaming the resources/views/vendor/backpack/base

**Comment by pxpm** [comment, marked as answer]
Thanks for the feedback @blondie63 very helpful and appreciated! 🙏 

I've merged https://github.com/Laravel-Backpack/docs/pull/499 with your suggestions. 

Let me know if I've missed something.

Cheers
